docs/source/docker-usage.md

```diff
@@ -2,11 +2,29 @@
 
 - [Docker Usage Guide](#docker-usage-guide)
   * [Basic Usage](#basic-usage)
+  * [Basic Usage for bats project](#basic-usage-for-bats-project)
   * [Docker Gotchas](#docker-gotchas)
   * [Extending from the base image](#extending-from-the-base-image)
   
 ## Basic Usage
 
+For test suites that are intended to run in isolation from the bats project, you can mount the test directory and run the [official bats docker image](https://hub.docker.com/r/bats/bats):
```

Comment:
In comparison to the original, this Changes the meaning. Yes, we want to document how to run tests in projects that are not Bats with the docker file. However, the text was referring to the third party code that should be tested.

```suggestion
For test suites that are intended to run in isolation from their project code, you can mount the test directory and run the [official bats docker image](https://hub.docker.com/r/bats/bats):
```